src/utils/sortPeople.ts

```diff
@@ -0,0 +1,34 @@
+import { Person } from '../types';
+
+export function sortPeople(
+  sort: string | null,
+  order: string | null,
+  people: Person[],
+) {
+  const sortedPeople = people;
```

Comment:
The variable `sortedPeople` is directly assigned to the input array `people`, which means the original array is being mutated. Consider creating a shallow copy of the array before sorting to avoid side effects: `const sortedPeople = [...people];`.